SPECS-EXTENDED/rubygem-rspec/rubygem-rspec.spec

```diff
@@ -1,11 +1,13 @@
+%global debug_package %{nil}
+%global gemdir %(IFS=: R=($(gem env gempath)); echo ${R[${#R[@]}-1]})
```

Comment:
Nit: we've recently added this macro to be provided by `ruby`, so this shouldn't be necessary anymore. It's fine to leave it there, most Ruby specs currently have it (we might clean them up at some point in the future), I just wanted to let you know for future reference.